src/pages/black-friday.tsx

```diff
@@ -38,72 +55,114 @@ const BlackFriday = ({ lang, metatagsDescriptions, langJson, navbarLang, footerL
       description: langJson.blackFriday.FeatureSection.feature4.subtitle1,
     },
   ];
+
+  const onBusinessPlansSelected = (isBusiness: boolean) => {
+    setIsBusiness(isBusiness);
+  };
+
+  const onCheckoutButtonClicked = (priceId: string, isCheckoutForLifetime: boolean) => {
+    const lifetimeSpacePlan = products?.individuals[Interval.Lifetime].find((product) => product.priceId === priceId);
```

Comment:
Different coupon codes are being used here for lifetime plans - is that the intention?